src/dot.rs

```diff
@@ -324,6 +324,174 @@ where
     }
 }
 
+#[cfg(feature = "dot_parser")]
+#[macro_use]
+pub mod dot_parser {
```

Comment:
nitpick: It'd be easier to read if this module was reflected in the file hierarchy (in a `src/dot/dot_parser.rs`).